src/EventListener/Dca/RendererDcaListener.php

```diff
@@ -125,20 +125,21 @@ public function getRenderSettings($dataContainer): array
 
         if ($dataContainer->activeRecord) {
             $repository = $this->repositoryManager->getRepository(LayerModel::class);
-            $layer      = $repository->find((int) $dataContainer->activeRecord->pid);
+            $layer      = $repository->find((int)$dataContainer->activeRecord->pid);
```

Comment:
```suggestion
            $layer      = $repository->find((int) $dataContainer->activeRecord->pid);
```

Please do not change the coding style here